src/assets/styles/10-settings/_colours.scss

```diff
@@ -24,8 +24,6 @@
   --colour-blue-10: #dbeaff;
   --colour-blue-05: #f0f6ff;
 
-  --colour-concrete-90: #292929;
```

Comment:
There were two variables for the same colour (#292929):
--colour-concrete-90 and --colour-grey-80

Reply:
👍  it's an older colour name so tnx for removing. Design system colours can be found here FYI
https://app.zeplin.io/project/5b4e06b48ae4580d4871178a/screen/5ea86a03c8963abb60b549cd